.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 - conda install --yes python=$TRAVIS_PYTHON_VERSION numpy scipy matplotlib sympy networkx
   nose h5py
 - pip install -i https://pypi.binstar.org/pypi/simple pygraphviz==1.3rc1
-- pip install python-coveralls
+- pip install python-coveralls pexpect
```

Comment:
Also pedantic: I think anaconda offers pexpect (see http://docs.continuum.io/anaconda/pkg-docs), so ideally we'd install that with conda rather than pip.